packages/appstore/src/constants/wallets-intro-content-config.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import WalletsImage from 'Assets/svgs/wallets';
+import { localize, Localize } from '@deriv/translations';
+
+type TWalletsIntroContent = {
```

Comment:
@aizad-deriv You don't need to define the type, Let it infer.

Reply:
Done 😃 👍🏼